code/game/objects/items/sharpener.dm

```diff
@@ -1,57 +1,77 @@
 		return
-	if(requires_sharpness && !I.get_sharpness())
+	if(requires_sharpness && !I.get_sharpness()) //You can't sharpen an item that isn't sharp with a whetstone that requieres items to be sharp
 		to_chat(user, "<span class='warning'>You can only sharpen items that are already sharp, such as knives!</span>")
 		return
-	if(is_type_in_list(I, list(/obj/item/melee/transforming/energy, /obj/item/dualsaber)))
+	if(is_type_in_list(I, list(/obj/item/melee/transforming/energy, /obj/item/dualsaber))) //You can't sharpen photons
 		to_chat(user, "<span class='warning'>You don't think \the [I] will be the thing getting modified if you use it on \the [src]!</span>")
 		return
 
-	var/signal_out = SEND_SIGNAL(I, COMSIG_ITEM_SHARPEN_ACT, increment, max)
-	if(signal_out & COMPONENT_BLOCK_SHARPEN_MAXED)
+	var/signal_out = SEND_SIGNAL(I, COMSIG_ITEM_SHARPEN_ACT, increment, max) //This is used to check more things if the item has a relevant component. As of December 2020, this is only used for two_handed and transforming.
+	if(signal_out & COMPONENT_BLOCK_SHARPEN_MAXED) //If the item's components enforce more limits on maximum power from sharpening,  we fail
 		to_chat(user, "<span class='warning'>[I] is much too powerful to sharpen further!</span>")
 		return
-	if(signal_out & COMPONENT_BLOCK_SHARPEN_BLOCKED)
+	if(signal_out & COMPONENT_BLOCK_SHARPEN_BLOCKED) //Checks for "other" restrictions
 		to_chat(user, "<span class='warning'>[I] is not able to be sharpened right now!</span>")
 		return
-	if((signal_out & COMPONENT_BLOCK_SHARPEN_ALREADY) || (I.force > initial(I.force) && !signal_out))
+	if((signal_out & COMPONENT_BLOCK_SHARPEN_ALREADY) || (I.force > initial(I.force) && !signal_out)) //No sharpening stuff twice
 		to_chat(user, "<span class='warning'>[I] has already been refined before. It cannot be sharpened further!</span>")
 		return
-	if(!(signal_out & COMPONENT_BLOCK_SHARPEN_APPLIED))
-		I.force = clamp(I.force + increment, 0, max)
-		I.wound_bonus = I.wound_bonus + increment
+	if(!(signal_out & COMPONENT_BLOCK_SHARPEN_APPLIED)) //If the item has a relevant component and COMPONENT_BLOCK_SHARPEN_APPLIED is returned, the item only gets the throw force increase
+		I.force = clamp(I.force + increment, 0, max) //We can't end up with a force better than max
+		I.wound_bonus = I.wound_bonus + increment //wound_bonus has no cap
 	user.visible_message("<span class='notice'>[user] sharpens [I] with [src]!</span>", "<span class='notice'>You sharpen [I], making it much more deadly than before.</span>")
 	playsound(src, 'sound/items/unsheath.ogg', 25, TRUE)
-	I.sharpness = SHARP_EDGED
-	I.throwforce = clamp(I.throwforce + increment, 0, max)
-	I.name = "[prefix] [I.name]"
-	name = "worn out [name]"
+	I.sharpness = SHARP_EDGED //When you whetstone something, it becomes an edged weapon, even if it was previously dull or pointy
+	I.throwforce = clamp(I.throwforce + increment, 0, max) //We can't end up with a throw force better than max
```

Comment:
This prob doesn't need to be called out, sharpness is a bit better but still kinda eh